Ginger/GingerCoreNET/ActionsLib/ActSetVariableValue.cs

```diff
@@ -134,6 +136,49 @@ public override void Execute()
                 {
                     ((VariableDynamic)Var).ValueExpression = this.Value;
                 }
+                else if (Var.GetType() == typeof(VariableNumber))
+                {
+                    try
+                    {
+                        var varNumber = ((VariableNumber)Var);
+                        if (float.Parse(calculatedValue) >= float.Parse(varNumber.MinValue) && float.Parse(calculatedValue) <= float.Parse(varNumber.MaxValue))
+                        {
+                            varNumber.Value = calculatedValue;
+                        }
+                        else
+                        {
+                            Status = Amdocs.Ginger.CoreNET.Execution.eRunStatus.Failed;
+                            Error = $"The value {calculatedValue} is not in the range, {Var.Name}:-[Min value:{varNumber.MinValue}, Max value:{varNumber.MinValue}]   {GingerDicser.GetTermResValue(eTermResKey.Variable) }.";
+                        }
+                    }
+                    catch (System.Exception ex)
+                    {
+                        Status = Amdocs.Ginger.CoreNET.Execution.eRunStatus.Failed;
```

Comment:
Need to update error field instead of Status

Reply:
changes pushed.